src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -872,8 +872,8 @@ private void CheckNewModifier(Symbol symbol, bool isNew, DiagnosticBag diagnosti
                             }
                             else if (((CSharpParseOptions)overridingMemberLocation.SourceTree?.Options)?.IsFeatureEnabled(MessageID.IDS_FeatureNullableReferenceTypes) == true &&
                                 (compilation = overridingMember.DeclaringCompilation) != null &&
-                                !overridingMemberType.Equals(overriddenProperty.TypeWithAnnotations,
-                                                             TypeCompareKind.AllIgnoreOptions & ~(TypeCompareKind.IgnoreNullableModifiersForReferenceTypes)))
+                                !compilation.Conversions.WithNullability(true)
```

Comment:
Expression repeated four times in the method. Consider caching to a local.

Reply:
It's cached in a field so perf shouldn't matter and I didn't think putting a local at the top was super useful since it would be so disconnected from the usage.